tests/unit/api/main.py

```diff
@@ -1822,12 +1836,12 @@ def test_show_node(self):
             'metadata': [
                 {
                     'label': 'EK',
-                    'value': 'pk',
+                    'value': json.dumps('pk'),
```

Comment:
You shouldn't need `dumps` here (or below); the `loads` should parse the whole structure, so you want `expected` to be the parsed python value, not the serialized JSON.